insights-client.spec

```
@@ -63,17 +63,72 @@ Resource Optimization service upon modifying ros_collect parameter to True.
 
 
 %build
-%{meson} \
-    -Dpython=%{__python3} \
-%if (0%{?rhel} && 0%{?rhel} < 10)
-    -Dredhat_access_insights=true \
-%endif
-    %{nil}
-%{meson_build}
+sed -e "s|@PYTHON@|%{__python3}|g" -e "s|@pythondir@|%{python3_sitelib}|g" \
+    src/insights-client.in > insights-client
 
+sed -e "s|@PACKAGE@|%{name}|g" \
+    -e "s|@PACKAGE_VERSION@|%{version}|g" \
+    -e "s|@CORE_SELINUX_POLICY@||g" \
+    src/insights_client/constants.py.in > src/insights_client/constants.py
 
 %install
-%{meson_install}
+
+# ./src/
+install -d -m 755 %{buildroot}%{_bindir}/
+install -m 755 insights-client %{buildroot}%{_bindir}/
+install -d -m 755 %{buildroot}%{python3_sitelib}/insights_client/
+cp -pr src/insights_client/* %{buildroot}%{python3_sitelib}/insights_client/
+rm -rf %{buildroot}%{python3_sitelib}/insights_client/tests
```

Adding the trailing slash is best practice for directories
```suggestion
rm -rf %{buildroot}%{python3_sitelib}/insights_client/tests/
```